uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -186,6 +186,20 @@ define([
           }
 
           return deferred.promise;
+        },
+
+        /**
+         * Override of parent method
+         *
+         * @param {object} result - The accumulator object
+         * @param {string} key - The property name
+         */
+        toAPIFilter: function (result, __, key) {
```

Comment:
The `_` or `__` as a function param indicates a param for which you have no use in the function implementation, but that you still need to declare since you need to get the parameter after that. I don't remember where I saw this before, but i learned that this is a convention. In fact I used `__` just to avoid the confusion with `_`